src/main/webapp/app/course/manage/course-update.component.html

```diff
@@ -1,19 +1,38 @@
 <div class="row justify-content-center">
     <div class="col-md-8">
         <form [formGroup]="courseForm" (ngSubmit)="save()" #formDir="ngForm">
-            <div class="headline">
+            <div class="container headline">
+                <input type="file" #fileInput style="display: none" (change)="setCourseImage($event)" />
                 @if (croppedImage) {
-                    <jhi-secured-image [src]="croppedImage" [cachingStrategy]="CachingStrategy.LOCAL_STORAGE" />
+                    <div class="image-with-icons">
+                        <jhi-secured-image class="secured-image" [src]="croppedImage" [cachingStrategy]="CachingStrategy.LOCAL_STORAGE" />
+
+                        <button type="submit" class="btn icon-button-wrapper icon-button-1" (click)="deleteCourseIcon()">
```

Comment:
Why "submit"? Can probably be "button", too

Reply:
Fixed